WordPress/Classes/ViewRelated/System/Sidebar/SiteMenuViewController.swift

```diff
@@ -43,7 +43,7 @@ final class SiteMenuViewController: UIViewController {
 private final class SiteMenuListViewController: BlogDetailsViewController {
     override func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
         let title = super.tableView(tableView, titleForHeaderInSection: section)
-        return title == nil ? (section == 0 ? 0 : 20) : 52
+        return title == nil ? 0 : 48
```

Comment:
Removing ghost sections in the new site menu. The app no longer uses the grouped style, so these extra vertical spacing made no sense. In the new menu, all items now fit vertically, even "Site Settings".